scrapper.py

```diff
@@ -44,20 +78,103 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(article_url, article_id)
+
+    def _fill_article_with_meta_information(self, article_bs):
+        date = article_bs.find('span', class_='date')
+        datetime_object = datetime.strptime(date.text, '%d.%m.%Y')
+        self.article.date = datetime_object
+
+        title = article_bs.find('h1')
+        self.article.title = title.text
+
+        sublink_bs = article_bs.find('div', class_='sublink')
+        author_bs = sublink_bs.find('a')
+        self.article.author = author_bs.text
+
+        topics_bs = sublink_bs.find_all('a')[1]
+        self.article.topics = topics_bs.text
+
+    def _fill_article_with_text(self, article_bs):
+        divs = article_bs.find('div', class_='memo')
+        ps = divs.find_all('p')
+        self.article.text = ''
+        for p in ps:
+            self.article.text += p.text
+
+    def parse(self):
+        response = requests.get(url=self.article_url)
+
+        article_bs = BeautifulSoup(response.text, 'lxml')
+
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+
+        return self.article
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    path = Path(base_path)
+    if path.exists():
+        shutil.rmtree(base_path)
+    path.mkdir(parents=True, exist_ok=True)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path) as file:
+        config = json.load(file)
+    max_articles = config["total_articles_to_find_and_parse"]
+    seed_urls = config["seed_urls"]
+    if not seed_urls:
+        raise IncorrectURLError
+
+    if not isinstance(seed_urls, list):
+        raise IncorrectURLError
+
+    if len(seed_urls) == 0:
```

Comment:
to check if a list is empty not only is it unnecessary to call len, but it is also extra work to compare it to 0